mautic/mautic
# Fixes campaign interval tests and fixes minor bug with scheduling

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Tests were failing again due to DST differences with Johannesburg which does not recognize DST

It also fixes a bug where if the scheduled time is earlier than the earliest allowed hour, it schedules for the same day at that hour. For example, if the "schedule at 10am" and it's 9am, the action should be scheduled 10am today rather than 10am tomorrow. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Run the tests
2. Create a campaign and schedule an action with a start hour of one hour past the time it is currently (use any end hour as long)
3. Execute the campaign and notice it'll be scheduled for tomorrow at that time

#### Steps to test this PR:
1. Run the tests
2. Repeat above and it should schedule for today at the start time
